database/engine/journalfile.c

```diff
@@ -275,6 +275,7 @@ static int check_journal_file_superblock(uv_file file)
 static void restore_extent_metadata(struct rrdengine_instance *ctx, struct rrdengine_journalfile *journalfile,
                                     void *buf, unsigned max_size)
 {
+    static uint64_t page_error_map[4] = {0,0,0,0};
```

Comment:
This is nice!

Since you have done this work, I suggest to put it in libnetdata, as:

```c

typedef struct bitmap256 {
   uint64_t data[4];
} BITMAP256;

extern bool bitmap256_get_bit(BITMAP256 *ptr);
extern void bitmap256_set_bit(BITMAP256 *ptr, bool bit);
```

So that we can use it elsewere too.